cfg/builder/builder_walk.cc

```diff
@@ -382,8 +382,7 @@ BasicBlock *CFGBuilder::walk(CFGContext cctx, ast::Expression *what, BasicBlock
                     auto blockLast = walk(cctx.withTarget(blockrv)
                                               .withBlockBreakTarget(cctx.target)
                                               .withLoopScope(headerBlock, postBlock, true)
-                                              .withSendAndBlockLink(link)
-                                              .withRubyBlockId(newRubyBlockId),
```

Comment:
checking my understanding: this `.with` isn't needed anymore because `walk` already gets a new `current` as one of its arguments?

Reply:
Yes, that's correct 👍